tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/statements/vardeclr/VarDeclrSemanticTest.java

```diff
@@ -80,6 +79,9 @@ public void testIncompleteVarDecl() {
         int indx = 0;
         validateError(result, indx++, "unknown type 'L'", 20, 26);
         validateError(result, indx++, "missing open parenthesis token", 21, 1);
+        validateError(result, indx++,
+                "incompatible types: expected '()', found 'typedesc<ballerina/lang.test:0.0.0:Caller>'",
+                21, 5);
```

Comment:
Can write these 2 lines in-line. 
```suggestion
                "incompatible types: expected '()', found 'typedesc<ballerina/lang.test:0.0.0:Caller>'", 21, 5);
```